README.md

```diff
@@ -2,7 +2,7 @@
 
 # School Library
 
-> School Library app: Implement the classes to represent students and teachers
+> School Library app: Decorator design pattern to correct the names of people
 
 
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused headers (`usage` `run tests` `deployment`), and add instructions to the `Getting started` section, so your documentation looks more organized